pxtcompiler/emitter/service.ts

```diff
@@ -18,15 +18,16 @@ namespace ts.pxtc {
         n: string;
         t: any;
         scale?: number;
+        snippet?: string;
     }
 
     export const ts2PyFunNameMap: pxt.Map<FunOverride> = {
-        "Math.trunc": { n: "int", t: ts.SyntaxKind.NumberKeyword },
-        "Math.min": { n: "min", t: ts.SyntaxKind.NumberKeyword },
-        "Math.max": { n: "max", t: ts.SyntaxKind.NumberKeyword },
-        "Math.abs": { n: "abs", t: ts.SyntaxKind.NumberKeyword },
-        "Math.randomRange": { n: "randint", t: ts.SyntaxKind.NumberKeyword },
-        "console.log": { n: "print", t: ts.SyntaxKind.VoidKeyword },
+        "Math.trunc": { n: "int", t: ts.SyntaxKind.NumberKeyword, snippet: "int(0)" },
+        "Math.min": { n: "min", t: ts.SyntaxKind.NumberKeyword, snippet: "min(0, 0)" },
+        "Math.max": { n: "max", t: ts.SyntaxKind.NumberKeyword, snippet: "max(0, 0)" },
+        "Math.abs": { n: "abs", t: ts.SyntaxKind.NumberKeyword, snippet: "abs(0)" },
+        "Math.randomRange": { n: "randint", t: ts.SyntaxKind.NumberKeyword, snippet: "randint(0, 10)" },
```

Comment:
Is randint special to us? I see only randomrange in python

Reply:
https://docs.python.org/3/library/random.html#random.randint

Reply:
Oh see. Cool thanks!